bin/tunnel-pydev-for-ipv6.sh

```
@@ -0,0 +1,2 @@
+apt-get update && apt-get install -y 6tunnel
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)